usr/lib/cca_stdll/cca_specific.c

```diff
@@ -6114,6 +6116,32 @@ CK_RV token_specific_aes_cbc(STDLL_TokData_t * tokdata,
     return rc;
 }
 
+static CK_BBOOL token_specific_filter_mechanism(STDLL_TokData_t *tokdata,
+                                                CK_MECHANISM_TYPE mechanism)
+{
+    CK_BBOOL rc = CK_FALSE;
+
+    switch(mechanism) {
+    case CKM_AES_XTS:
+    case CKM_AES_XTS_KEY_GEN:
+#ifndef NO_PKEY
+         if (ccatok_pkey_option_disabled(tokdata)) {
```

Comment:
I guess you should also check `cca_private->pkey_wrap_supported` here. 
Otherwise we report the XTS mechs to be available, although we don't have a CCA card that supports protected keys.....